Assets/MixedRealityToolkit.Tests/PlayModeTests/BoundsControlTests.cs

```diff
@@ -0,0 +1,314 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using Microsoft.MixedReality.Toolkit.UI;
+using NUnit.Framework;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Linq;
+using Assert = UnityEngine.Assertions.Assert;
+using Microsoft.MixedReality.Toolkit.UI.Experimental;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
```

Comment:
To match the experimental structure elsewhere, please put under Assets/MixedRealityToolkit.Tests/Experimental/PlayModeTests/BoundsControlTests.cs and namespace Microsoft.MixedReality.Toolkit.Tests.Experimental, or similar if there's a structure you'd prefer. But it should include Experimental in folder and namespace.